src/gpt-auto-generator/gpt-auto-generator.c

```diff
@@ -895,8 +895,16 @@ static int parse_proc_cmdline_item(const char *key, const char *value, void *dat
 
         assert(key);
 
-        if (proc_cmdline_key_streq(key, "systemd.gpt_auto") ||
-            proc_cmdline_key_streq(key, "rd.systemd.gpt_auto")) {
+        if (STR_IN_SET(key, "fstab", "rd.fstab")) {
```

Comment:
Hmm, I'm not sure about the duplicate code, especially when the actual performance gain is negligible IMO.

Reply:
okay, I will drop this part.